lib/between_meals/repo/hg.rb

```diff
@@ -144,6 +152,17 @@ def name
         nil
       end
 
+      def status
+        s = Mixlib::ShellOut.new(
+          "#{@bin} status",
+          :cwd => File.expand_path(@repo_path)
+        ).run_command
+        s.error!
+        s.stdout
+      rescue
+        nil
```

Comment:
The point is returning with nil in case of non-zero code, instead of whatever happens to be in s.stdout